requirements/default.txt

```diff
@@ -87,7 +87,7 @@ gevent==1.4.0 \
 gunicorn==19.9.0 \
     --hash=sha256:aa8e0b40b4157b36a5df5e599f45c9c76d6af43845ba3b3b0efe2c70473c2471 \
     --hash=sha256:fa2662097c66f920f53f70621c6c58ca4a3c4d3434205e608e121b5b3b71f4f3
-# TODO: Update to redis[hiredis]>=2.10.6 or later, and move hiredis to constraints
+# TODO: Update to redis[hiredis]>=3.0, and move hiredis to constraints
```

Comment:
I misunderstood the sequence of ``setup.py`` changes, redis 3.0 is the first version that supports ``hiredis`` as an extra, and will be updated as part of #874.

The [redis 3.3.0 release notes](https://github.com/andymccurdy/redis-py/blob/d811ae71dbdbeeb4fc0ee73a96b7fbdb1aec8522/CHANGES#L38-L40) include this line:

> * Added support for hiredis-py 1.0.0 encoding error support. This should make the PythonParser and the HiredisParser behave identically when encountering encoding errors. Thanks Brian Candler.

This suggests the hiredis update should wait for at least redis 3.3.0